pyshelf/search/metadata.py

```diff
@@ -1,11 +1,15 @@
-from elasticsearch_dsl import String, Nested, Boolean, DocType
+from elasticsearch_dsl import String, Nested, Boolean, DocType, tokenizer, analyzer
+
+
+# Required for case sensitivity
+metadata_analyzer = analyzer("metadata_analyzer", tokenizer=tokenizer("keyword"))
```

Comment:
Cool added to name and clarified with new analyzer name - 7a07a89d59250127fce21b5f1b68492046b3eb60